CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ include_directories(
 find_package(Eigen3 REQUIRED)
 include_directories(${EIGEN3_INCLUDE_DIR})
 
+# add boost support for file system
+find_package(Boost REQUIRED COMPONENTS filesystem system)
```

Comment:
I would like to avoid any dependency on Boost

Reply:
Sure, I will make the change to use std::filesystem instead. Can I change "set(CMAKE_CXX_STANDARD 14)" to 17?

Reply:
We will have to take a look at whether that is realistic for where we want to deploy it. We can talk more about what you need, and as this only needs to work on Linux we may just do something less portable in the short term.

Reply:
Looks like he is mostly using it in main.cpp so that you can run the histogram stuff from the C++ executable.

@zhuokaizhao, we have mostly been using python and pybind11 to run the underlying C++ functions rather than running them in the `main()` C++ function. Most of our examples are either [ipython-notebooks](https://github.com/OpenChemistry/stempy/blob/master/examples/stem_image.ipynb) or [python scripts](https://github.com/OpenChemistry/stempy/blob/master/examples/electron_stem_image.py). It might be good for you to write a histogram example in python rather than in C++, so it is easier to deal with filesystems.

Reply:
I think @zhuokaizhao was using main.cpp for testing the algorithm, now that we have the python binds in place, we should just add a notebook or script, we can do any filesystem things there.